website/other/bezier-rs-demos/src/utils/options.ts

```diff
@@ -21,3 +21,19 @@ export const tMinimumSeperationOptions = {
 	step: 0.001,
 	default: 0.05,
 };
+
+export const intersectionErrorOptions = {
+	variable: "error",
+	min: 0.001,
+	max: 0.525,
+	step: 0.0025,
+	default: 0.02,
+};
+
+export const intersectionMinimumSeperationOptions = {
+	variable: "minimum_seperation",
```

Comment:
Small typo: separation vs. seperation. Might be easiest to just ctrl-f for seperation and replace all instances, because I've noticed it a few other places.